ui/pages/TraceListPage.tsx

```diff
@@ -8,7 +8,13 @@ export default function TraceListPage() {
       .then(res => res.json())
       .then(setTraces);
   }, []);
-  return (
+  return traces.length === 0 ? (
+    <div className="m-8 flex-col space-y-4">
```

Comment:
Is not including the `flex` class as in the other case intentional?

Reply:
Ah I had been trying to prevent this from happening:

![1](https://user-images.githubusercontent.com/1863868/213569211-0a28e9be-3e86-413d-b98c-788257724cfe.png)

Which could be done by removing `flex-col` instead of `flex`. But now I realize I should prevent this from happening as well:

![2](https://user-images.githubusercontent.com/1863868/213569409-10294ba8-ad75-4e02-9c08-07e0eccfb3fe.png)

Since it's the only thing on the page when the page is empty, how do you feel about removing everything except for `m-8`?

Reply:
I didn't actually check what it looked like, I just noticed the difference and wanted to double check it. Whatever works and looks good is fine.

Reply:
Ok, done!